sdk/python/tests/integration/registration/test_universal_registry.py

```diff
@@ -59,7 +60,7 @@
 def local_registry() -> Registry:
     fd, registry_path = mkstemp()
     registry_config = RegistryConfig(path=registry_path, cache_ttl_seconds=600)
-    return Registry("project", registry_config, None)
+    return Registry("project", registry_config, None, auth_config=NoAuthConfig())
```

Comment:
I am a bit concerned about the backward compatibility, can we keep the default value of auth_config to NoAuthConfig if is not initialized by default? If we roll out the new version of feast with the rbac changes is it going to break existing functionality on top of feast code if there is such thing exists?